assembl/static2/js/app/components/common/goUp.jsx

```diff
@@ -1,47 +1,59 @@
+// @flow
 import React from 'react';
 import { Translate } from 'react-redux-i18n';
+import classnames from 'classnames';
+
 import { withScreenHeight } from './screenDimensions';
 
-class GoUp extends React.Component {
-  constructor(props) {
-    super(props);
-    this.displayButton = this.displayButton.bind(this);
-    this.state = { isHidden: true, position: 'fixed' };
-  }
+type Props = {
+  screenHeight: number
+};
+
+type State = {
+  isHidden: boolean,
+  position: string
+};
 
-  componentWillMount() {
+class GoUp extends React.Component<Props, State> {
+  state = { isHidden: true, position: 'fixed' };
+
+  componentDidMount() {
     window.addEventListener('scroll', this.displayButton);
   }
 
   componentWillUnmount() {
     window.removeEventListener('scroll', this.displayButton);
   }
 
-  displayButton() {
+  displayButton = () => {
     const footer = document.getElementById('footer');
     if (!footer) {
       return;
     }
     const footerHeight = footer.offsetHeight;
     const { screenHeight } = this.props;
+    // $FlowFixMe document.body is not null
```

Comment:
add the control on `document.body !== null` instead of $FlowFixMe ?

Reply:
it's not necessary :)